### DISCUSSION TITLE
Can we automatically open the project pane and terminal when I open Zed/project?

### DISCUSSION THREAD

**Discussion by NeroCoreNull** [zed-industries/zed#11605] [Feature Requests]
I tried to find some documentation about it but I was unable to. It would be nice for the terminal to open on the bottom and the project panel on the left when I open zed/open projects. If I have missed a setting please let me know!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
